config.json

```diff
@@ -115,47 +118,47 @@
       ]
     },
     {
-      "slug": "list-ops",
+      "slug": "list_ops",
       "uuid": "903374d2-8155-4fe3-bf90-5b6359c7b5cc",
       "core": false,
       "unlocked_by": null,
       "difficulty": 1,
       "topics": null
     },
     {
-      "slug": "scrabble-score",
+      "slug": "scrabble_score",
       "uuid": "5b44ccd4-b1f4-4b86-89c8-f1dec5905ccb",
       "core": false,
       "unlocked_by": null,
       "difficulty": 1,
       "topics": null
     },
     {
-      "slug": "word-count",
+      "slug": "word_count",
       "uuid": "9460b65d-dc80-4a95-8782-b395d2cc979e",
       "core": false,
       "unlocked_by": null,
       "difficulty": 1,
       "topics": null
     },
     {
-      "slug": "two-fer",
+      "slug": "two_fer",
       "uuid": "3ecc2d1c-55e0-45c9-ba35-57d7d8cfd51e",
       "core": false,
       "unlocked_by": null,
       "difficulty": 1,
       "topics": null
     },
     {
-      "slug": "atbash-cipher",
+      "slug": "atbash_cipher",
       "uuid": "82169e85-3938-48da-ada9-7394adadc57a",
       "core": false,
       "unlocked_by": null,
       "difficulty": 1,
       "topics": [
-        "cipher",
-        "string"
+        "string",
+        "cipher"
       ]
     }
   ]
-}
+}
```

Comment:
I don't think it matters that there is no newline at the end, but I am sure that configlet will tell us if it is a problem.

Reply:
Yay, this is now the only difference between the `config.json` files.

Since configlet doesn't seem to care, and this is not a file that we as humans actually have to read (since we have the sweet schemification) are you fine with not having a trailing newline?  or should I make it write an extra line at the end?

Reply:
Weird. It feels a bit too buggy to have. If we don't care about how the output looks anyway, why bother formatting the json output at all?

Reply:
Well, that symbol will go away once this is merged, because the diff will be comparing against a file with no trailing newline.

The reason why I still support formatting the json:
Occasionally core members of exercism can review how a track is doing (not like a work review just checking on the the health of the track).  I don't want to scare them with unformatted `json`, that might make them initially think we have dropped the ball entirely.

To my knowledge we would be the only track without pretty printed json.  While individuality doesn't bother me I do think about the consequences of too much obscurity in a community environment.

Reply:
Also, contributors to other tracks might try a couple exercises, and they are more likely to come check our track out/look for an issue to turn into a PR.  Even though the json won't be fundamental to our track it is likely to alienate or give the wrong impression when folks not so immersed in the project come to visit our code base.

Reply:
I am ready to resolve this conversation if we are in agreement?